activestorage/app/models/active_storage/blob.rb

```diff
@@ -402,6 +404,12 @@ def service_metadata
       end
     end
 
+    def touch_attachment_records
+      attachments.includes(:record).each do |attachment|
+        attachment.record.touch
+      end
```

Comment:
My goal is that when the ActiveStorage::Blob changes (which has no updated_at column), the corresponding attachments' records change to invalidate cache.

The associations are currently:

```ruby
class ActiveStorage::Blob < ActiveStorage::Record
  has_many :attachments
end

ActiveStorage::Attachment < ActiveStorage::Record
  belongs_to :record, polymorphic: true, touch: true
  belongs_to :blob, class_name: "ActiveStorage::Blob", autosave: true
end
```

Any time the Attachment changes, it touches `record`. So perhaps I should just make the Blob touch all of its Attachments and let the Attachments touch the `record`s? This change would make my test pass. It does feel a bit weird that right now we can have the Blob change, the `record` change, but not the Attachment.

```suggestion
      attachments.includes(:record).each do |attachment|
        attachment.touch
      end
```

Reply:
Yeah, I think you should rely on that `touch: true` by just calling `attachment.touch`.

Reply:
Sounds good